cloudkeeper-core/cloudkeeper-dsl/src/main/java/com/svbio/cloudkeeper/dsl/DSLPluginDescriptor.java

```diff
@@ -15,7 +15,11 @@
  * <p>An instance of this class provides the {@link Class} instances (as specified by the DSL) that make up a
  * CloudKeeper plug-in declaration.
  */
-public class DSLPluginDescriptor {
+public final class DSLPluginDescriptor {
+    private DSLPluginDescriptor() {
+        throw new AssertionError(String.format("No %s instances for you!", getClass().getName()));
+    }
+
```

Comment:
I meant only the new private constructor, which is a non-sensical addition. Making the class final is reasonable.